### DISCUSSION TITLE
idea can not login Github copilot

### DISCUSSION THREAD

**Discussion by yangfanchuhai** [community/community#52815] [Copilot Conversations]
I have config my copilot in github settings,but when i login to github in idea  toos>Github Copilot > Login to Github is error, the error msg is "9:57 PM	GitHub Copilot: Sign in failed. Reason: Request signInInitiate failed with message: Unexpected token N in JSON at position 0, request id: 5, error code: -32603
" what's the problem?

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @yangfanchuhai , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
